### PR TITLE
sys-process/nmon: Bump version

### PR DESCRIPTION
**Pull request by ntnn**
Package-Manager: Portage-2.3.6, Repoman-2.3.1

### CODE REVIEW

**Comment by ntnn** [comment]
Ping at my mentor @Polynomial-C

File: sys-process/nmon/metadata.xml [hunk 1]
```diff
@@ -1,8 +1,14 @@
-<?xml version='1.0' encoding='UTF-8'?>
+<?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-<!-- maintainer-needed -->
-	<upstream>
-		<remote-id type="sourceforge">nmon</remote-id>
-	</upstream>
```
Comment (Polynomial-C) [inline sys-process/nmon/metadata.xml]: Why did you remove the <upstream> tag and its content?

**Comment by ntnn** [comment]
On 2017-08-02  7:03, Lars Wendler wrote:
> Polynomial-C commented on this pull request.
>
> >  <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
>  <pkgmetadata>
> -
> -	<upstream>
> -		<remote-id type="sourceforge">nmon</remote-id>
> -	</upstream>
>
> Why did you remove the <upstream> tag and its content?
>
> --
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/gentoo/gentoo/pull/5269#pullrequestreview-53713500

That was unintentionally, I'm amending that commit.

--
/"\  ASCII Ribbon Campaign
\ /  - against HTML emails
 X   - against proprietory attachments
/ \  http://en.wikipedia.org/wiki/ASCII_Ribbon_Campaign
